src/containers/trends/CBGSlicesAnimationContainer.js

```diff
@@ -141,13 +140,11 @@ export default class CBGSlicesAnimationContainer extends React.Component {
   render() {
     const { mungedData } = this.state;
     if (_.isEmpty(mungedData)) {
-      const { margins, svgDimensions } = this.props;
-      const xPos = (svgDimensions.width / 2) - margins.left + margins.right;
-      const yPos = (svgDimensions.height / 2) - margins.top + margins.bottom;
       return (
-        <text className={styles.noDataMsg} id="noDataMsg" x={xPos + 40} y={yPos}>
-          No CGM data for this time period :(
-        </text>
+        <NoData
```

Comment:
Please remove the rendering of `NoData` from this component entirely. The task here was to move the rendering up a level so that it's being handled in common for both cbg and smbg versions of the trends view in `TrendsSVGContainer`.  It looks like that's been done, so cleanup here please :)